server/src/main/java/org/candlepin/async/tasks/ExportJob.java

```diff
@@ -170,12 +171,12 @@ public void validate() throws JobConfigValidationException {
                 String consumerUuid = arguments.getAsString(CONSUMER_KEY);
                 String cdnLabel = arguments.getAsString(CDN_LABEL);
 
-                if (!(consumerUuid instanceof String) || ((String) consumerUuid).isEmpty()) {
+                if (consumerUuid == null || consumerUuid.isEmpty()) {
```

Comment:
:+1:

The instanceof check was leftover from before the switch to the type-safe JobArguments. Thanks for updating this.